### ISSUE TITLE
name finder

### ISSUE THREAD

**Issue by arden446** [arden446/playground#7]
In testerthing.html it says _str.toUpperCase();_ isn't a function even though when set up the same scenario in about:blank it works as it should. Why is this?

**Comment by daniel-jonathan** [comment]
Was this the Array issue, where you were setting it to an Array?

**Comment by arden446** [comment]
No, this was the one when needed to use the .map function to loop through using .to uppercase.

**Comment by arden446** [comment]
Hey!✌️ Download a new cool game Flip the Gun! It's hilarious!!!😂😂😂 Download Flip the Gun FOR FREE right now!🔥🔥🔥😍👉
https://itunes.apple.com/app/id1333542838


Sent from my iPhone

**Comment by daniel-jonathan** [comment]
Hey! Looks wild. I will try it.
